modules/oc-mirror-dry-run-v2.adoc

```diff
@@ -15,19 +15,22 @@ Verify your image set configuration by performing a dry run without mirroring an
 +
 [source,terminal]
 ----
-$ oc mirror -c <image_set_config_yaml> --from file://<oc_mirror_workspace_path> docker://<mirror_registry_url> --dry-run --v2
+$ oc mirror -c <image_set_config_yaml> file://<oc_mirror_workspace_path> --dry-run --v2
 ----
++
 Where:
```

Comment:
[Docs guidelines section on how to format description lists](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#:~:text=of%20requested%20CPU.-,If%20you%20must,-provide%20additional%20information), I'm going to make several comments to these lines based on that section
```suggestion
where:
```